src/cascadia/TerminalCore/TerminalDispatch.hpp

```diff
@@ -38,12 +38,12 @@ class TerminalDispatch : public Microsoft::Console::VirtualTerminal::TermDispatc
 private:
     ::Microsoft::Terminal::Core::ITerminalApi& _terminalApi;
 
-    static bool s_IsRgbColorOption(const ::Microsoft::Console::VirtualTerminal::DispatchTypes::GraphicsOptions opt) noexcept;
-    static bool s_IsBoldColorOption(const ::Microsoft::Console::VirtualTerminal::DispatchTypes::GraphicsOptions opt) noexcept;
-    static bool s_IsDefaultColorOption(const ::Microsoft::Console::VirtualTerminal::DispatchTypes::GraphicsOptions opt) noexcept;
+    static constexpr bool s_IsRgbColorOption(const ::Microsoft::Console::VirtualTerminal::DispatchTypes::GraphicsOptions opt) noexcept;
+    static constexpr bool s_IsBoldColorOption(const ::Microsoft::Console::VirtualTerminal::DispatchTypes::GraphicsOptions opt) noexcept;
+    static constexpr bool s_IsDefaultColorOption(const ::Microsoft::Console::VirtualTerminal::DispatchTypes::GraphicsOptions opt) noexcept;
 
     bool _SetRgbColorsHelper(const std::basic_string_view<::Microsoft::Console::VirtualTerminal::DispatchTypes::GraphicsOptions> options,

```

Comment:
oh no, had we messed this one up too? with the missing `&`?

Reply:
Sorry, what? Which one? It's not clear what you're talking about from this comment.

Reply:
We talked verbally. This is OK.